Automattic/wp-calypso
# Make placement of JITMs to be more specific

#### Changes proposed in this Pull Request

This PR makes the message path used for JITMs be more specific, to allow targeting of specific paths in the interface.

Also see: 1153012803416853-as-1157265372777921

#### Testing instructions



* Create a JITM targeting `plugins-upload`.
* On Calypso, navigate to plugins -> install.
* Verify test JITM appears.

Fixes #30491
